src/dash.ts

```diff
@@ -33,4 +34,35 @@ export class Dash {
 
         return '';
     }
-}
\ No newline at end of file
+
+    /**
+     * Returns the command to handle the URI in the command line.
+     *
+     * @param {string} OS - string representing the OS
+     * @return {string} name and flags of the command that takes the URI as an argument
+     */
+    getDashURIHandler(OS: string): string {
+        // hacky switch statement
+        return {
+            'darwin': 'open -g',
+            'linux': 'zeal',
+            // start Zeal before we pass it a URI so it works whether Zeal is running in background or not.
+            // Why is this needed? I have no idea, but I think Qt has to start up a listener on cmd before it will work.
+            'win32': 'start dash-plugin:// && start'
+        }[OS] || 'zeal';
+    }
+
+    /**
+     * Returns the OS specific URI to be handled.
+     *
+     * @param {string} uri - original constructed URI
+     * @param {string} OS - string representing the OS
+     * @return {string} OS-specific URI to pass to handler, mainly because of Windows
+     */
+    getOSSpecificURI(uri: string, OS: string): string {
+        return {
+            // on Windows, start can't deal with the double quotes, and & needs to be escaped with ^
+            'win32': uri.replace('&', '^&')
+        }[OS] || '"' + uri + '"';
+    }
```

Comment:
My rationale behind passing the OS as a param was to make it clearer that this was something that changed the return value of the functions, and also because I wanted to pass the OS to the function in the tests. Also makes it easier to debug in future just in case, but using a class variable works too. 

I don't mind either way, but ultimately it's your extension, so which style would you prefer to have?